docs/use/gateway.md

```diff
@@ -33,6 +33,19 @@ If you want the settings to be kept upon gateway restart, you can publish the co
 Auto discovery is enable by default on release binaries, on platformio (except for UNO). With Arduino IDE please read the [advanced configuration section](../upload/advanced-configuration#auto-discovery) of the documentation.
 :::
 
+# Firmware update from MQTT (ESP only)
+
+The gateway can be updated through an MQTT message by providing a JSON formatted message with a version number, OTA password (optional, see below), and URL to fetch the update from.  
+
+To enable this functionality, `MQTT_HTTPS_FW_UPDATE` will need to be defined or the line that defines in in user_config.h will need to be uncommented.
+
+::: tip
+If using an unsecure MQTT broker it is **highly recommended** to disable the password checking by setting the macro `MQTT_HTTPS_FW_UPDATE_USE_PASSWORD` to 0 (default is 1 (enabled)), otherwise a clear text password may be sent over the network.
+:::
+
+### Example firmware update message:
+`mosquitto_pub -t firmware_update -m "{\"version\":\"version_tag\",\"password\":\"OTAPASSWORD\",\"url\":\"https://github.com/1technophile/OpenMQTTGateway/releases/download/v0.9.6/esp32-m5stack-ble-firmware.bin\"}"`
```

Comment:
I would remove the backslash from the MQTT message I'm not sure we need them and replace version_tag by test, 
```json
{
  "version": "test",
  "password": "OTAPASSWORD",
  "url": "https://github.com/1technophile/OpenMQTTGateway/releases/download/v0.9.6/esp32-m5stack-ble-firmware.bin"
}
```

The example topic could be presented as:
```
home/<gateway_name>/commands/firmware_update
```

Reply:
Ok, will do.

Reply:
done